packages/ckeditor5-mention/docs/_snippets/examples/chat-with-mentions.html

```diff
@@ -119,6 +119,10 @@
 		border-top-right-radius: 0;
 	}
 
+	.chat .chat__editor.highlighted + .ck.ck-editor .ck-content {
```

Comment:
I see that you probably did this `+` selector hack because the editor engine was overriding the CSS class that you set on the editing root straight away. You can get rid of the hack, check my later comment.

Reply:
Actually, I just followed the convention. 

```
.chat .chat__editor + .ck.ck-editor {
    margin-top: 0;
}

.chat .chat__editor + .ck.ck-editor .ck.ck-toolbar {
    border-top-left-radius: 0;
    border-top-right-radius: 0;
}
```

I wasn't sure why `+` was used in existing styles and assumed there must be some reason for this, i.ex. to avoid the collisions if more editors would be added on the same page.

But indeed, I also didn't want to mess up with the content of the editor.